app/src/main/java/org/p2p/wallet/debug/torus/DebugTorusFragment.kt

```diff
@@ -33,7 +33,11 @@ class DebugTorusFragment : BaseFragment(R.layout.fragment_debug_torus) {
 
             networkServicesUrlProvider.loadTorusEnvironment().apply {
                 environmentTextView.text = baseUrl
-                verifierTextView.text = verifier
+                verifierTextView.text = if (subVerifier.isNullOrBlank()) {
+                    verifier
+                } else {
+                    "verifier = $verifier\nsubVerifier = $subVerifier"
+                }
```

Comment:
it's always to put explicitly that there is subVerifier here instead of making it gone
```suggestion
                verifierTextView.text = "verifier = $verifier\nsubVerifier = ${subVerifier ?: "-"}"
```
nothing wrong with view looking like this
verifier = smth
subVerifier = -